component/provider.jsonnet

```diff
@@ -304,4 +304,56 @@ local controllerConfigRef(config) =
       role,
       rolebinding,
     ],
+  [if params.providers.minio.enabled then '10_provider_minio']:
+    local provider = params.providers.minio;
+
+    local sa = kube.ServiceAccount(provider.controllerConfig.serviceAccountName) {
+      metadata+: {
+        namespace: provider.namespace,
+      },
+    };
+    local role = kube.ClusterRole('crossplane:provider:provider-minio:system:custom') {
+      rules: [
+        {
+          apiGroups: [ 'minio.crossplane.io' ],
+          resources: [ '*' ],
+          verbs: [ 'get', 'list', 'watch', 'update', 'patch', 'create', 'delete' ],
+        },
+        {
+          apiGroups: [ '' ],
+          resources: [ 'secrets' ],
+          verbs: [ 'get', 'list', 'watch', 'create', 'watch', 'patch', 'update', 'delete' ],
```

Comment:
it's just me, but can we ensure the same order in both rows? so there are `get, list, watch, update, patch (...)` - it's easier to read diff later and find missing permissions